Arduino-Bling/README.md

```diff
@@ -0,0 +1,76 @@
+# Integrating NeoPixel with Arduino & Robot Code
+This directory hosts the source code to interface roborio/wpilib robot fuctionality with
+NeoPixel/Arduino.
+
+## High-level code flow
+1. Robot code sets the desired LED blink pattern
+2. LED subsystem sends the required blink pattern over the serial port
+   to the Arduino
+3. Arduino code frequently checks the serial port to determine if a new
+   blink pattern is required --> if so, it changes the blink pattern
+
+## How to interface robot code w/ Arduino for LED animation
+Robot code interfaces to Arduino via serial port communication using
+UART protocol, implemented as a CDC/ACM class communication device over
+USB. While not "classic" UART, it operates the same. Serial object in
+this case only refers to UART communications. The commands to change
+LED animation pattern is sent as an octet byte over the serial.
+
+Robot code:
+* `LED` subsystem:
+  * specifies the blink patterns --> see `LED::BlinkState`
```

Comment:
I think that this should be "specifies the desired animations". The LED subsystem doesn't actually define the patterns themselves (ie, what colors are put up in what pattern). That is all internal to the Arduino, and the animation ID (handle?) is all that is being specified. Correct?

Reply:
This is correct -- recommended revision is accurate description, tx!